test/extended/syncer/syncer.go

```diff
@@ -0,0 +1,76 @@
+package syncer
+
+import (
+	"os"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area/transparent-multi-cluster]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-syncer")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][BYO] Validate create, modifying, deleting a deployment from KCP gets synced to the pcluster", func() {
+		pclusterKubeconfig := os.Getenv("PCLUSTER_KUBECONFIG")
+		if pclusterKubeconfig == "" {
+			g.Skip("No pcluster kubeconfig set for the test scenario")
+		}
+		k.SetupWorkSpaceWithNamespace()
+		myWs := k.WorkSpace()
+		k.SetGuestKubeconf(pclusterKubeconfig)
+
+		g.By("# Create workload sync and generate syncer resources manifests in current workspace")
+		mySyncer := NewSyncTarget()
+		mySyncer.OutputFilePath = "/tmp/" + myWs.Name + "." + mySyncer.Name + ".yaml"
+		mySyncer.Create(k)
+		defer mySyncer.Clean(k)
+
+		g.By("# Apply syncer resources on pcluster and wait for synctarget become ready")
+		defer k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("delete").Args("-f", mySyncer.OutputFilePath).Execute()
+		err := k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("apply").Args("-f", mySyncer.OutputFilePath).Execute()
+		o.Expect(err).ShouldNot(o.HaveOccurred())
+		mySyncer.WaitUntilReady(k)
+		mySyncer.CheckDisplayAttributes(k)
+
+		g.By("# Create workload using the shared compute should work well")
+		myDeployment := exutil.NewDeployment(exutil.SetDeploymentNameSpace(myWs.Namespaces[0]))
+		defer myDeployment.Clean(k)
+		myDeployment.Create(k)
+		myDeployment.WaitUntilReady(k)
+		myDeployment.CheckDisplayAttributes(k)
+
+		g.By("# Check the deployment's status on pcluster")
+		avaiablableReplicasOnPcluster, getError := k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("get").Args("deployment", "-A", `-o=jsonpath={.items[?(@.metadata.name=="`+myDeployment.Name+`")].status.availableReplicas}`).Output()
```

Comment:
```suggestion
		avaiablableReplicasOnPcluster, getError := k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("get").Args("deployment", myDeployment.Name, `-o=jsonpath={.status.availableReplicas}`).Output()
```
Is it better to directly specify the deployment name as the suggestion?